pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -2114,6 +2114,22 @@ let
         };
       };
 
+      hbenl.vscode-test-explorer = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "vscode-test-explorer";
+          publisher = "hbenl";
+          version = "2.21.1";
+          sha256 = "sha256-fHyePd8fYPt7zPHBGiVmd8fRx+IM3/cSBCyiI/C0VAg=";
+        };
+        meta = {
+          license = lib.licenses.mit;
```

```suggestion
          license = lib.licenses.mit;
```
Please follow alphabetical order in the attrs.
Meta fields aren't alphabetically ordered, because broken is always last. (It will work here.)

---

I could not directly commit this suggestion, but i applied it in 825a2b6